Doc/library/time.rst

```diff
@@ -120,13 +120,6 @@ An explanation of some terminology and conventions is in order.
 
 The module defines the following functions and data items:
 
-.. data:: altzone
```

Comment:
Thank you.  I had originally removed it, but then put it back in because I left some of the new data constants (starting with time.CLOCK_HIGHRES.) in that section.  I didn't know if I should move the newer data items to the bottom with the deprecated ones or not.  Maybe I should change the one on the bottom to be 'Deprecated data items:'?